lib/wrapper.js

```diff
@@ -145,7 +149,7 @@ class RequestWrapper {
       return JSON.parse(response.body);
     } catch (ex) {
       logger.fromError('fatal_error', ex, this._getLogParameters());
-      throw new SuiteRequestError(ex.message, 500);
+      throw new SuiteRequestError(ex.message, 500, response.body);
```

Comment:
Does `response.body` never contain sensitive data on error? If we're not sure about it, I won't add it to the error object.

Reply:
In other cases it is added to the error...in this case the message is failed to parse as JSON and skipped passing the body leaving no way to find out the reason of the failure.